src/modules/site-v2/base/views/strains.py

```diff
@@ -147,7 +147,25 @@ def isotype_page(isotype_name, release=None):
   if not isotype_strains:
     abort(404)
 
-  # Fetch isotype images
+  return render_template('strain/isotype.html', **{
+    "title": f"Isotype {isotype_name}",
+    "isotype": isotype_strains,
+    "isotype_name": isotype_name,
+    "isotype_ref_strain": [x for x in isotype_strains if x.isotype_ref_strain][0],
+    "strain_json_output": dump_json(isotype_strains)
+  })
+
+#
+# Isotype image URLs
+#
+@strains_bp.route('/isotype-img/<isotype_name>/')

```

Comment:
API design -- prefer resource and related components: eg: `/isotype/<isotype_name>img`, if available.